packages/contracts/contracts/core/HackMoneyVault.sol

```diff
@@ -128,6 +131,7 @@ contract HackMoneyVault is Multicall, Ownable, BaseVault {
 
         strategy.setBoard(boardId);
 
+        // FIXME: Do we have tests for this?
```

Comment:
Not really, once we have the script with the fork we can test the whole work flow for one week